Short-Answer/Algorithms_Answers.md

```diff
@@ -2,9 +2,9 @@
 
 ## Exercise I
 
-a) This one is a O(log n). As the size of n increases, the runtime or space used will grow at a alil bit of a slowwer rate
+a) This one is a O(1). As the size of n increases, the runtime or space used will grow at a alil bit of a slowwer rate
 
-b)This one is a O(n^c) As the size of the input increases, the space used will grow at a faster rate;
+b)This one is a O(n) As the size of the input increases, the space used will grow at a faster rate;
 
 c)This one is a O(n) as the size of the input increases, the runtime or space used is will grow at the same rate
```

Comment:
Check back on some videos about time complexity. 1 of these three are correct, but your explanations don't show a solid understanding of time complexities.